scripting/get5_apistats.sp

```diff
@@ -196,13 +212,28 @@ static void CheckForLogo(const char[] logo) {
   }
 }
 
-static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode status,
+static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode statusCode,
```

Comment:
You can change the data argument to `DataPack` instead of `int`, that way you won't have to cast it below.

Reply:
done